packages/checkbox-input/index.test.js

```diff
@@ -9,11 +9,14 @@ const defaultProps = getTestValues(props)
 defaultProps.field.onChange = () => null
 
 describe('<CheckboxInput />', () => {
-  it('should render a `.g-checkbox-input` <div> root element', () => {
-    const { container } = render(<CheckboxInput {...defaultProps} />)
+  it('should pass a provided className to the root element', () => {
+    const className = 'my-special-checkbox'
+    const { container } = render(
+      <CheckboxInput {...defaultProps} className={className} />
+    )
     const rootElem = container.firstChild
     expect(rootElem.tagName).toBe('DIV')
-    expect(rootElem).toHaveClass('g-checkbox-input')
+    expect(rootElem).toHaveClass(className)
```

Comment:
As mentioned in the `changeset`, and similar to many other component conversions to CSS modules, we no longer set  a global `g-{component-name}` class on the root element. However, to support [existing overrides](https://sourcegraph.hashi-mktg.com/search?q=g-checkbox-input&patternType=literal), we now support a `className` prop.